am-lib/src/main/java/uk/gov/hmcts/reform/amlib/AccessManagementService.java

```diff
@@ -20,6 +21,7 @@
 
 import static uk.gov.hmcts.reform.amlib.enums.Permission.READ;
 
+@SuppressWarnings("PMD")
```

Comment:
Remove please. Happy to have these on test classes but not on public ones. It literally disables PMD which is way too much.

Reply:
👍